src/test/ui/nll/maybe-initialized-drop.stderr

```diff
@@ -0,0 +1,46 @@
+error[E0506]: cannot assign to `x` because it is borrowed (Ast)
```

Comment:
So I can't seem to get the ui tests to output the Mir/NLL output. If I add the revisions comment, it fails to even recognize this file and considers every error a new error that wasn't expected. Removing the revisions but keeping the borrowck and nll compiler flags outputs this file. I tried reading the markdown file for tests, but it doesn't say revisions are supported for ui tests. What should I do here?

Reply:
Wait, why do you want/have revisions in this test? Also, I feel like these tests are a good example of where I would prefer a `compile-fail` test, at least until such time as the `ui` tests permit (and enforce) `//~ ERROR` annotations. In a compile-fail test, we could put (in the test itself) `//~ ERROR (Ast)` for where the old borrow checker reports an error and `//~ ERROR (Mir)` for where the new borrow checker -- with NLL enabled -- reports an error. The latter ought to I guess be a subset of the former. It's easier to read this "inline" than to read the stderr.

Reply:
(In particular, this test is about testing *that* errors occur, not necessarily what we emit when they do.)

Reply:
But I guess I'm missing the point. The point is that you only see `(Ast)` errors here, right?

Reply:
In which case it seems like a bug in the branch =)

Reply:
Sorry, I completely misunderstood the output of what was going on. I thought certain errors were only showing up when I had the revisions comment, but it's actually the `-Z nll` flag that silences the Mir errors. I'll need to look into this more. (My other confusion was that adding in the revisions comment cause the test to ignore the .stderr file, the expected errors section comes out blank, I thought they were related).